core/src/main/scala/codesearch/core/index/LanguageIndex.scala

```diff
@@ -107,12 +98,7 @@ trait Sources[VTable <: DefaultTable] {
     }
   }
 
-  def downloadFile(srcURL: String, dstFile: File): Unit = {
-//    s"curl -o ${dstFile.getPath} $srcURL" !!
-  }
-
-  def runCsearch(searchQuery: String,
-                 insensitive: Boolean, precise: Boolean, sources: Boolean): Array[String] = {
+  def runCsearch(searchQuery: String, insensitive: Boolean, precise: Boolean, sources: Boolean): Array[String] = {
```

Comment:
Why not put parameters in a column here, like you've done with archiveDownloadAndExtract?